### PR TITLE
fix(ui): Give up loading team avatar

### PR DESCRIPTION
**Pull request by scttcper**
If the team does not exist, returns null instead of loading forever

### CODE REVIEW

**Review by evanpurkhiser — APPROVED** [review]

File: static/app/components/avatar/actorAvatar.tsx [hunk 1]
```diff
@@ -31,9 +31,9 @@ function LoadTeamAvatar({
   ...props
 }: {teamId: string} & Omit<React.ComponentProps<typeof TeamAvatar>, 'team'>) {
   const {teams, isLoading} = useTeamsById({ids: [teamId]});
-  const team = teams.find(t => t.id === teamId);
+  const team = teams.find(t => t.id === teamId) ?? null;
```
Comment (evanpurkhiser) [inline static/app/components/avatar/actorAvatar.tsx:34]: Do you need the `?? null`. Won't it return `undefined`? is there a difference here

**Review by ryan953 — APPROVED** [review]
Looks like `<TeamAvatar>` will handle the null just fine.

File: static/app/components/avatar/actorAvatar.tsx (hunk 1 above)
Comment (ryan953) [reply static/app/components/avatar/actorAvatar.tsx:34]: https://github.com/getsentry/sentry/blob/28a38c7e5b412c53da4eae0d432575846992c16b/static/app/components/avatar/teamAvatar.tsx#L5-L7

**Comment by scttcper** [comment]
to be clear this was the previous behavor, i made it load forever. isError also is not returned since the api returns a 200 + empty array

**Pull request merged**